python/TestHarness/testers/RunApp.py

```diff
@@ -114,7 +114,9 @@ def getCommand(self, options):
       specs['cli_args'].append('Outputs/print_perf_log=true')
 
     if options.colored == False:
-      specs['cli_args'].append('--no-color')
+      specs['cli_args'].append('--color off')
+    else:
+      specs['cli_args'].append('--color on')
```

Comment:
@dschwen is correct - to preserve prev/normal test color behavior, we now need to force color on for tests.  Auto would be off.